ios/brave-ios/Sources/BraveVPN/Components/Settings/BraveVPNContactFormViewController.swift

```diff
@@ -317,14 +318,12 @@ class BraveVPNContactFormViewController: TableViewController {
   }
 
   private var getNetworkType: String {
-    let status = Reach().connectionStatus()
-
-    switch status {
-    case .offline, .unknown:
+    let type = Reachability.shared.status.connectionType
+    if type == .offline || type == .other {
```

Comment:
`other` is considered a satisfied connection based on the ReachabilityType initializer, wouldn't it make more sense to just use `type.description` for all types?

Reply:
Fixed.